cfme/intelligence/chargeback/rates.py

```diff
@@ -355,6 +355,9 @@ class ComputeRateEdit(CFMENavigateStep):
     VIEW = EditComputeChargebackView
     prerequisite = NavigateToSibling('Details')
 
+    def am_i_here(self):
```

Comment:
same ^^

Reply:
Removed am_i_here methods, modified is_displayed of EditComputeChargebackView and EditStorageChargebackView to include self.save_button.is_displayed. To make it work, I had to change Button(title='Save Changes') to Button('Save'), because when the button is greyed out, it has no title attribute.